docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -876,9 +876,9 @@ Review `results_summary.log` file under artifacts dir location. Make sure that o
 To receive scalability benchmark results for four-node Jira DC with app-specific actions:
 
 1. Scale your Jira Data Center deployment to 3 nodes as described in [Run 4](#run4).
-1. Check Index is synchronized to new nodes the same way as in [Run 4](#run4).
-1. Scale your Jira Data Center deployment to 4 nodes as described  in [Run 4](#run4).
-1. Check Index is synchronized to new nodes the same way as in [Run 4](#run4).
+1. Check Index is synchronized to the new node #3 the same way as in [Run 4](#run4).
+1. Scale your Jira Data Center deployment to 4 nodes as described in [Run 4](#run4).
+1. Check Index is synchronized to new node #4 the same way as in [Run 4](#run4).
```

Comment:
Please add "the" after "to", Like in ...user-guide-confluence.md

Reply:
done